cmd/data-update-tracker.go

```diff
@@ -366,6 +364,8 @@ func (d *dataUpdateTracker) deserialize(src io.Reader, newerThan time.Time) erro
 		return err
 	}
 	switch tmp[0] {
+	case 1:
+		return errors.New("dataUpdateTracker: deprecated data version, updating.")
```

Comment:
does this migrate existing data? @klauspost because existing data usage info cannot be changed or what was calculated. 

As either bucket quota or prometheus would depend on this data. So is the mcs like UI..

Reply:
No, existing caches will be dropped. This will force a re-scan on upgrade.

But since we need the full paths instead of hashes we need that anyway.

Reply:
This is not the JSON data.

Reply:
@harshavardhana